x-pack/test/functional/es_archives/visualize/default/data.json

```diff
@@ -125,26 +125,8 @@
 {
   "type": "doc",
   "value": {
-    "id": "custom-space:index-pattern:metricbeat-*",
```

Comment:
this is a corrupt saved object because the _id is invalid (`custom-space` should be `custom_space`) which means it won't serialize and probably was never used so instead of trying to fix it I just deleted it.